src/tests/system/tests/test_identity.py

```diff
@@ -547,36 +511,138 @@ def test_identity__lookup_idmapping_of_posix_and_non_posix_user_and_group(client
-        4. Start SSSD
+        1. Create a group
+        2. Create the following users 'user_same_gid', 'user_diff_gid' and 'user_group_gid'.
+        Set the uids and gids to match the condition defined by in the username
+        3. Configure SSSD with "ldap_id_mapping = false" and "auto_private_groups = false" and start SSSD
     :steps:
-        1. getent passwd "user_same"
-        2. getent passwd "user_different"
+        1. Lookup 'user_same_gid' and look up the user's gid
+        2. Lookup 'user_diff_gid' and look up the user's gid
+        3. Lookup 'user_group_gid' and look up the user's gid
     :expectedresults:
-        1. Uid equals to gid
-        2. Uid does not equal to gid
+        1. The user is found with the configured values, and the group is *NOT* found
+        2. The user is found with the configured values, and the group is *NOT* found
+        3. The user is found with the configured values, and the group is found
     :customerscenario: True
     :requirement: IDM-SSSD-REQ: SSSD can automatically create user private groups for users
     """
-    provider.user("user_same").add(uid=111111, gid=111111)
-    provider.user("user_different").add(uid=111111, gid=100000)
+    provider.group("group").add(gid=444444)
+    provider.user("user_same_gid").add(uid=111111, gid=111111)
+    provider.user("user_diff_gid").add(uid=222222, gid=333333)
+    provider.user("user_group_gid").add(uid=444444, gid=444444)
+
+    client.sssd.domain["auto_private_groups"] = "false"
+    client.sssd.domain["ldap_id_mapping"] = "false"
+
+    client.sssd.start()
+
+    assert client.tools.getent.passwd("user_same_gid").gid == 111111
+    # IPA manages auto_private_groups on the server and is true by default
+    if isinstance(provider, IPA):
+        assert client.tools.getent.group(111111)
+    else:
+        assert not client.tools.getent.group(111111)
+
+    assert client.tools.getent.passwd("user_diff_gid").gid == 333333
+    assert not client.tools.getent.group(333333)
+
+    assert client.tools.getent.passwd("user_group_gid").gid == 444444
+    assert client.tools.getent.group(444444)
+
+
+@pytest.mark.ticket(bz=1695577)
+@pytest.mark.importance("high")
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_identity__lookup_when_auto_private_groups_is_set_to_hybrid(client: Client, provider: GenericProvider):
+    """
+    :title: Look up users when auto private groups is set to hybrid
+    :description: When set to hybrid, if the uid and gid match for the user, it will act as if a
+    group exists similar to when the parameter is set to true. Like when it's set to false, if the
+    group exists in LDAP, it will look at whatever is existing for its values.
+    :setup:
+        1. Create a group
+        2. Create the following users 'user_same_gid', 'user_diff_gid', 'user_no_gid' and 'user_group_gid'.
+        Set the uids and gids to match the condition defined by in the username
```

Comment:
Please add `3.`  or is it a part of setup `3.` then please add indentation, alien under `Create`

Reply:
It's apart of 2, indented.